launcher

```diff
@@ -4,6 +4,16 @@ command=$1
 config=$2
 opt=$3
 
+# Docker doesn't like uppercase characters, spaces or special characters, catch it now before we build everything out and then find out
+re='[A-Z/ !@#$%^&*()+~`=]'
+if [[ $config =~ $re ]];
+  then
+    echo 
+    echo "ERROR: Config name must not contain upper case characters, spaces or special characters. Correct config name and rerun $0."
```

Comment:
I think its ok for now, but we should sweep through launcher and output all errors to stderr